client/layout/sidebar/style.scss

```diff
@@ -1,731 +1,206 @@
-		flex-direction: column;
-		flex: 1;
-
-		> li:last-child {
-			margin-bottom: 20px;
-		}
-	}
-
-	.sidebar__menu {
-		.gridicon {
-			display: none;
-		}
-
-		svg,
-		.gridicons-my-sites,
-		.gridicons-chevron-down,
-		.gridicons-external {
-			display: inline-block;
-		}
-
-		.selected svg {
-			fill: var( --sidebar-menu-selected-a-color );
-		}
+		padding: 10px 16px 10px 20px;
+		font-weight: 600;
+		cursor: pointer;
 
-		&.is-togglable {
-			.sidebar__menu-list {
-				padding-bottom: 0;
-				padding-top: 0;
-			}
-		}
+		&:hover {
+			background-color: var( --sidebar-menu-hover-background );
+			color: var( --sidebar-menu-hover-color );
 
-		&.is-toggle-open {
-			.sidebar__heading {
-				background-color: transparent;
+			.sidebar__menu-icon,
+			.sidebar__expandable-arrow {
+				fill: var( --sidebar-menu-hover-color );
 			}
 		}
 	}
 
-	.sidebar__button {
-		display: none;
-	}
-
-	.sidebar__menu-list a {
-		padding: 10px 16px 10px 55px;
-	}
-
-	.is-togglable .sidebar__heading {
-		font-weight: 600;
-		justify-content: flex-start;
-	}
-
-	.is-togglable .sidebar__heading span {
-		flex: 0 1 100%;
-	}
-
-	.sidebar__menu.is-togglable .sidebar__heading,
-	.sidebar__menu a {
-		font-size: 15px;
-		font-weight: 600;
-	}
-
-	.sidebar__menu-list .sidebar__menu-link-text {
-		font-weight: normal;
-	}
-
-	.is-togglable .sidebar__heading .gridicons-chevron-down,
-	.sidebar .is-togglable .sidebar__heading .material-icon-exit_to_app {
-		margin-right: 0;
-		top: 0;
-	}
-
-	.sidebar__menu.wp-admin .sidebar__heading .gridicon {
-		order: 0;
+	.sidebar__expandable-title {
+		flex: 1 1 0;
 	}
 
-	.sidebar__heading .gridicons-chevron-down {
-		align-self: flex-end;
-		order: 99;
+	.sidebar__expandable-arrow {
+		transition: transform 0.1s linear;
+		fill: var( --sidebar-gridicon-fill );
 	}
 
-	.is-togglable.is-toggle-open .sidebar__heading .gridicon:not( .gridicons-chevron-down ) {
-		transform: rotate( 0deg );
+	.sidebar__menu-link {
+		padding-left: 55px;
 	}
 
-	/* WP Admin Link */
-
-	.sidebar__menu.sidebar__wp-admin {
-		width: 100%;
-		margin-top: auto;
-
-		@include breakpoint( '<660px' ) {
-			border: none;
+	&.is-toggle-open {
+		.sidebar__expandable-arrow {
+			transform: rotate( 180deg );
 		}
 	}
+}
 
-	.sidebar__menu.sidebar__wp-admin .wp-admin {
-		border-top: 1px solid var( --sidebar-border-color );
-		justify-content: flex-start;
+// Sometimes we show info at the bottom of the sidebar. For
+// example on My Site(s), for users with a single site, we
+// show a button to add a new site.
+.sidebar__wp-admin,
+.sidebar__footer {
+	margin-top: 16px;
+	padding-top: 16px;
+	border-top: 1px solid var( --sidebar-border-color );
+}
+
+// This seems to add a border to the bottom of every
+// .sidebar__menu (whatever that is). I guess to make
+// the menu look better on phones?
+@include breakpoint( '<660px' ) {
+	li.sidebar__menu {
+		border-bottom: 1px solid var( --sidebar-border-color );
 
-		@include breakpoint( '<660px' ) {
-			border-top: none;
```

Comment:
I'm not so sure about getting rid of this, it seems to currently prevent there being a double border on mobile.

<img width="344" alt="Screenshot 2019-08-28 at 08 52 59" src="https://user-images.githubusercontent.com/43215253/63836364-54a38200-c971-11e9-9bb0-1003788f03f2.png">

But with this branch, there is a double border, and I'm assuming that's not intentional. 

<img width="349" alt="Screenshot 2019-08-28 at 08 52 53" src="https://user-images.githubusercontent.com/43215253/63836381-608f4400-c971-11e9-80c4-e79c3c03266d.png">

Reply:
I decided to remove these mobile-only borders. It's hard to make them right in all scenarios, and they are not terribly useful. Other popular mobile apps (looked at Slack and Instagram) don't have borders on menu items and still are great.

Reply:
Agreed, it doesn't seem necessary and it still looks good.